src/Nri/Ui/Button/V6.elm

```diff
@@ -63,7 +63,7 @@ import Nri.Ui.AssetPath as AssetPath exposing (Asset)
 import Nri.Ui.Colors.Extra as ColorsExtra
 import Nri.Ui.Colors.V1 as Colors
 import Nri.Ui.Fonts.V1
-import Nri.Ui.Icon.V4 as Icon exposing (IconType)
+import Nri.Ui.Icon.V5 as Icon exposing (IconType)
```

Comment:
Ooo wait we can't do this, can we? Won't this change will break all our existing uses of Nri.Ui.Button.V6?

I ran `elm diff`, and it says this is fine, which seems suspicious to me. I'm going to investigate a bit more.

Reply:
Yeah it looks like it's this: https://github.com/elm/compiler/issues/1777

Reply:
Ouch, good catch! I'll change that back to how it was before.